web/beacon-app/src/application/api/ApiService.ts

```diff
@@ -15,30 +15,22 @@ axiosInstance.defaults.withCredentials = true;
 // intercept request and check if token has expired or not
 axiosInstance.interceptors.request.use(
   async (config: any) => {
-    const token = getCookie('bc_atk');
-    const csrfToken = getCookie('csrf_token');
-    const decodedToken = token && decodeToken(token);
-    if (decodedToken) {
-      const { exp } = decodedToken;
-      const now = new Date().getTime() / 1000;
-      if (exp < now) {
-        //console.log('token expired');
-        // refresh token
-
-        // refreshToken();
-        //const accessToken = getCookie('bc_atk');
-        // const { data } = await QuarterDeckAuth.refreshToken(refreshToken);
-        // setCookie('bc_atk', data.access_token, { expires: data.expires_in });
-        // setCookie('bc_rtk', data.refresh_token, { expires: data.expires_in });
-        // setCookie('csrf_token', data.csrf_token, { expires: data.expires_in });
-        //config.headers.Authorization = `Bearer ${accessToken}`;
-        return config;
+    // since the server set the cookie with httpOnly flag
+    // it will be sent automatically by the browser
+    const bearer = config.headers.Authorization;
+    const token = bearer ? bearer.split(' ')[1] : null;
+    if (token) {
+      const decodedToken = decodeToken(token) as any;
+      if (decodedToken) {
+        const { exp } = decodedToken;
+        const now = new Date().getTime() / 1000;
+        if (exp < now) {
+          // token expired so logout user and clear cookies
+          // we could refresh token later on
+          // logout();
```

Comment:
Should `logout()` be commented out here? It appears as if nothing is happening in lines 28-30.

Reply:
Due to the fact that we already have a logout flow in the Sidebar component, this is for a future update. The comment was added to remind us that we will need to refactor that from the expiration date in the future.

Reply:
I think it could be helpful to add  a `TODO` here with some of the wording from your reply above for others reading the code. Also, to be sure, is nothing executed inside the `if` statement on lines `27-31`?